### PR TITLE
Add basic documentation to Robot classes

### PR DESCRIPTION
**Pull request by JordanMartinez**

### CODE REVIEW

File: subprojects/testfx-core/src/main/java/org/testfx/robot/DragRobot.java [hunk 1]
```diff
@@ -22,15 +22,40 @@
...
+    /**
+     * Moves the mouse to {@link PointQuery#query()} before releasing the mouse.
+     *
+     * @param pointQuery the pointQuery
+     */
     public void dropTo(PointQuery pointQuery);
 
+    /**
+     * Moves the mouse horizontally by {@code x} and vertically by {@code y} before releasing the mouse.
+     * @param x the amount by which to move the mouse horizontally
```
Comment (brcolow) [inline subprojects/testfx-core/src/main/java/org/testfx/robot/DragRobot.java]: Missing space between description and `@param` tags (guess our extensive checkstyle rules doesn't check that, heh).

Comment (JordanMartinez) [inline subprojects/testfx-core/src/main/java/org/testfx/robot/DragRobot.java]: Fixed.
Yeah, I was wondering if there was any check style rules for that...

**Comment by brcolow** [comment]
Is this still a WIP? If so, let me know when you are done.

**Comment by JordanMartinez** [comment]
Sorta.... BaseRobot still needs javadoc, but most of its methods are called by the other Robot interfaces, thus making it redundant. However, there are 3 methods that can be implemented: `retrieveMouse()`, `captureRegion()`, and `awaitEvents()`. So, I'm working on those right now.

**Comment by JordanMartinez** [comment]
K. That should do it.

**Comment by brcolow** [comment]
Merged. Thanks a lot.

**Pull request merged**